.github/workflows/test.yml

```diff
@@ -847,9 +850,12 @@ jobs:
     needs: [setup]
     env:
       # Various feature flags
+      FF_TFM_PAYMENT_RECONCILIATION_ENABLED: 'true'
       FF_FEE_RECORD_CORRECTION_ENABLED: 'true'
       FF_TFM_DEAL_CANCELLATION_ENABLED: 'true'
       FF_PORTAL_FACILITY_AMENDMENTS_ENABLED: 'true'
+      FF_TFM_SSO_ENABLED: 'false'
```

Comment:
@AlexBramhill I have changed this to `false` as it is failing all the FF E2E tests, you will have to set it to `true` in your final PR please.